app/components/trainees/confirmation/diversity/view.rb

```diff
@@ -0,0 +1,76 @@
+module Trainees
+  module Confirmation
+    module Diversity
+      class View < GovukComponent::Base
+        include SanitizeHelper
+
+        attr_accessor :trainee
+
+        def initialize(trainee:)
+          @trainee = trainee
+        end
+
+        def get_diversity_information_rows
+          rows = [
+            {
+              key: "Diversity information",
+              value: I18n.t("components.confirmation.diversity.diversity_disclosure.#{trainee.diversity_disclosure}"),
+              action: govuk_link_to('Change<span class="govuk-visually-hidden"> diversity information</span>'.html_safe,
+                                    edit_trainee_diversity_disclosure_path(trainee.id)),
+            },
+          ]
+
+          if trainee.diversity_disclosure == "diversity_disclosed"
+            rows << {
+              key: "Ethnicity",
+              value: get_ethnic_group_content,
+              action: govuk_link_to('Change<span class="govuk-visually-hidden"> ethnicity</span>'.html_safe,
+                                    edit_trainee_diversity_ethnic_group_path(trainee.id)),
+            }
+
+            rows << {
+              key: "Disability",
+              value: tag.p(get_disability_selection, class: "govuk-body") + get_selected_disability_options,
+              action: govuk_link_to('Change<span class="govuk-visually-hidden"> disability</span>'.html_safe,
+                                    edit_trainee_diversity_disability_disclosure_path(trainee.id)),
+            }
+          end
+          rows
+        end
+
+        def get_ethnic_group_content
+          value = I18n.t("components.confirmation.diversity.ethnic_groups.#{trainee.ethnic_group}")
+
+          if trainee.ethnic_background.present? && trainee.ethnic_background != "Not provided"
```

Comment:
could we use the value from the enum? eg `!= Diversities::NOT_PROVIDED_VALUE` as this is the same enum value which is offered as an option on the form.